lib/mix/tasks/hexweb/check_names.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Mix.Tasks.Hexweb.CheckNames do
+  use Mix.Task
+  require Logger
+  alias HexWeb.Package
+  import Ecto.Query, only: [from: 2]
+
+  @shortdoc "Check package names for typosquatters"
+
+
+  def run(_args) do
+    Mix.Task.run "app.start"
+
+    threshold = Application.get_env(:hex_web, :jaro_threshold)
+
+    threshold
+    |> find_candidates
+    |> send_mail(threshold)
+
+    :ok
+  end
+
+  def new_packages do
+    from(p in Package, select: p.name, where: fragment("inserted_at >= CURRENT_DATE"))
+    |> HexWeb.Repo.all
+  end
+
+
+  def current_packages do
+    from(p in Package, select: p.name, where: fragment("inserted_at < CURRENT_DATE"))
+    |> HexWeb.Repo.all
+  end
+
+
+  def calculate_distances do
+    for new <- new_packages(), curr <- current_packages() do
```

Comment:
We should also compare new packages against each other.

`for new <- new_packages(), all <- all_packages(), new != all do`